src/appkit/menu/menu.rs

```diff
@@ -19,10 +19,10 @@ impl Menu {
     ///
     /// This method effectively does three things:
     ///
-    ///     - Consumes the MenuItem Vec, and pulls out handlers we need to cache
```

Comment:
I think the indents here made the docstring think it was rust? It was confusing to me.